ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -32,19 +36,23 @@ export default function CollectiblesItems({
   previouslyOwnedCollection = {},
 }) {
   const collectionsKeys = Object.keys(collections);
+  const collectiblesDropdownState = useSelector(getCollectiblesDropdownState);
+  const previousCollectionKeys = usePrevious(collectionsKeys);
 
-  // if there is only one collection present set it to open when component mounts
-  const [dropdownState, setDropdownState] = useState(() => {
-    return collectionsKeys.length === 1
-      ? {
-          [PREVIOUSLY_OWNED_KEY]: false,
-          [collectionsKeys[0]]: true,
-        }
-      : { [PREVIOUSLY_OWNED_KEY]: false };
-  });
```

Comment:
I guess these are arrays: `previousCollectionKeys`, `collectionsKeys`. Equality check below may not be useful.